packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.styles.ts

```diff
@@ -4,15 +4,15 @@ const COMMAND_BAR_HEIGHT = 44;
 
 export const getStyles = (props: ICommandBarStyleProps): ICommandBarStyles => {
   const { className, theme } = props;
-  const { palette } = theme;
+  const { semanticColors } = theme;
 
   return {
     root: [
       theme.fonts.medium,
       'ms-CommandBar',
       {
         display: 'flex',
-        backgroundColor: palette.white,
+        backgroundColor: semanticColors.bodyBackground,
```

Comment:
Are the updates to semantic colors being done in stages? It looks like a couple of the components are being converted to using semantic but not all.

Reply:
Yep! We are not slotting all of them because there are some gaps in which some conversions need a bit. Some slots need to be adjusted/added. So slotted these based on recommendations from Sirena and left the ones still referencing palette based on gaps she noticed.